apps/block_scout_web/assets/js/lib/analytics.js

```diff
@@ -0,0 +1,257 @@
+      return 'allAccounts'
+    case path.includes('/blocks') && path.includes('/transactions'):
+      return 'blockTransactions'
+    case path.includes('/blocks') && path.includes('/signers'):
+      return 'blockSigners'
+    case path.includes('/address') && path.includes('/transactions'):
+      return 'addressDetails'
+    case path.includes('/address') && path.includes('/token_transfers'):
+      return 'addressTokenTransfers'
+    case path.includes('/address') && path.includes('/tokens'):
+      return 'addressTokens'
+    case path.includes('/address') && path.includes('/internal_transactions'):
+      return 'addressInternalTransactions'
+    case path.includes('/address') && path.includes('/coin_balances'):
+      return 'addressCoinHistory'
+    case path.includes('/address') && path.includes('/logs'):
+      return 'addressLogs'
+    case path.includes('/address') && path.includes('/signed'):
+      return 'addressDowntime'
+    case path.includes('/address') && path.includes('/validations'):
+      return 'addressBlocksValidated'
+    case path.includes('/address') && path.includes('/celo'):
+      return 'addressCeloInfo'
+    // TODO: Add rest of contract pages
+    case path.includes('/address') && path.includes('/contracts'):
+      return 'contractAddressCode'
+    case path.includes('/address') && path.includes('/read_contract'):
+      return 'readContract'
+    case path.includes('/tx') && path.includes('/internal_transactions'):
+      return 'transactionInternalTransaction'
+    case path.includes('/tx') && path.includes('/logs'):
+      return 'transactionLogs'
+    case path.includes('/tx') && path.includes('/raw_trace'):
+      return 'transactionRawTrace'
+    case path.includes('/tx') && path.includes('/token_transfers'):
+      return 'transactionTokenTransfers'
+    default:
+      return path
+  }
+}
+
+// returns referrer path, strips domain name from referrer
+function getReferrerPath () {
+  const referrer = document.referrer
+  return referrer.replace('https://explorer.celo.org', '').replace('http://localhost:4000', '')
+}
+
+// returns relevant entity ID: Address, Transaction, Block, or Search Parameter
+function getEntityId () {
+  const pathSegments = window.location.pathname.split('/')
+  for (var i = 0; i < pathSegments.length; i++) {
+    if (pathSegments[i].includes('search')) {
+      return window.location.search.slice(3)
+    } else if (pathSegments[i] === 'blocks' && pathSegments[i + 1]) {
+      return pathSegments[i + 1]
+    } else if (pathSegments[i].slice(0, 2) === '0x') {
+      return pathSegments[i]
+    }
+  }
+}
+
+// returns blockscout network: Mainnet, Alfajores or Baklava
+function getNetwork () {
+  switch (window.location.host) {
+    case 'explorer.celo.org':
+      return 'mainnet'
+    case 'alfajores-blockscout.celo-testnet.org':
+      return 'alfajores'
+    case 'baklava-blockscout.celo-testnet.org':
+      return 'baklava'
+    default:
+      return 'mainnet'
```

Comment:
Maybe it's not a safe default. We have other, instance-specific hostnames, like `https://baklava-blockscout3.celo-testnet.org/` that would fall into this bucket and will be incorrectly attributed to mainnet.